wezterm-gui/src/glyphcache.rs

```diff
@@ -745,6 +779,46 @@ impl<T: Texture2d> GlyphCache<T> {
         self.block_sprite(block)
     }
 
+    fn box_drawing_sprite(&mut self, box_drawing: BoxDrawingKey) -> anyhow::Result<Sprite<T>> {
```

Comment:
This is a stripped version of `fn block_sprite`.

I can see two ways forward to merge the two:

1. Nest `BlockKey` into `BoxDrawingKey` as they serve the same purpose and use the same logic. Having said that, it does appear that box drawing and block elements are distinct from each other in the Unicode specification, so this might not be what we'd want.
2. Add a top-level `CustomDrawingGlyph` (or differently named) enum that contains both `BoxDrawing` and `Block`, which would allow us to have a single `fn custom_drawing_sprite` to avoid the duplication.

There are other ways to do this, but I didn't want to go in any specific direction yet without knowing your preference.

Reply:
I'd vote for adding renaming `BlockKey` `CustomGlyphKey` and then adding `LightVertical` to that enum.